src/server/status_server.rs

```diff
@@ -240,6 +339,7 @@ impl StatusServer {
                             (Method::GET, "/status") => Box::new(ok(Response::default())),
                             (Method::GET, "/pprof/profile") => Self::dump_prof_to_resp(req),
                             (Method::GET, "/config") => Self::config_handler(config.clone()),
+                            (Method::GET, "/rsperf/profile") => Self::dump_rsperf_to_resp(req),
```

Comment:
please keep the same with Go style, the path is `/debug/pprof/profile?seconds=30`